OpenRA.Mods.Cnc/ImportTiberianDawnLegacyMapCommand.cs

```diff
@@ -0,0 +1,209 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using OpenRA.FileFormats;
+using OpenRA.Mods.Common.UtilityCommands;
+using OpenRA.Primitives;
+
+namespace OpenRA.Mods.Cnc.UtilityCommands
+{
+	class ImportTiberianDawnLegacyMapCommand : IUtilityCommand
```

Comment:
Sounds good in theory, but won't work here in practice as key points in between are not very similar. I also don't want to change too much to avoid breaking things.